.github/workflows/namd.yml

```diff
@@ -8,12 +8,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: build-charm++
-      run: ./build charm++ netlrts-linux-x86_64 -g -j4 --with-production
-    - name: build-namd
+    - name: install-prerequisites
       run: |
         sudo apt-get update
         sudo apt-get -y install sfftw-dev
+        # test build without Fortran support
+        sudo apt-get -y remove gfortran
+    - name: build-charm++
+      run: ./build LIBS netlrts-linux-x86_64 -g -j4 --with-production
```

Comment:
Why is this building LIBS for the NAMD test?

Reply:
To verify that LIBS, including AMPI, build correctly without Fortran. This only increases the build time from 16m to 19m.

Reply:
Gotcha, sounds good.